feature-libs/customer-ticketing/occ/config/default-occ-customer-ticketing-config.ts

```diff
@@ -0,0 +1,11 @@
+import { OccConfig } from '@spartacus/core';
+
+export const defaultOccCustomerTicketingConfig: OccConfig = {
+  backend: {
+    occ: {
+      endpoints: {
+        getTicketDetails: 'users/${customerId}/tickets/${ticketId}',
```

Comment:
OCC endpoint is named "getTicket".  Please update to synchronize with backend

https://github.tools.sap/cx-commerce/commerce-apis/blob/develop/commerce-apis-internal/Artifacts/customerticketing-internal/customerticketing-internal.yaml#L6108

Reply:
got it